### PR TITLE
#2056 removing on-demand certificate creation

### PR DESCRIPTION
**Pull request by stenington**
This PR gets things to a state where certificates are shown to users through simpler controller queries without attempting to generate certificates on demand, and it temporarily removes certificates from the admin (#2060 will put them back a little differently).

### CODE REVIEW

**Comment by stenington** [comment]
@adrichey Do you feel like I should remove the CertificateButton Vue app in this PR?

**Comment by stenington** [comment]
@adrichey I think my remaining work here is to remove the CertificateButton app, and to remove the certificate override model field and endpoint. No current judge has an override set, so we're not losing data. There's a `drop_lowest_scores.rb` rake task we may need to think of a way to account for, but I'd rather do something like marking scores as dropped and then including them in the judge certificate level calculation than doing the override.

Just keeping you up to date.

**Comment by adrichey** [comment]
I think removing the CertificateButton and the endpoint makes sense at this point. Good to know that no one has an override right now. I think that marking the scores as dropped and then including them in the judge level calculation sounds like the right approach as well. Thanks for working on this Mike!

**Comment by adrichey** [comment]
@stenington Can you please rebase QA into your branch by running `git rebase origin/qa`? I tried to run it locally to get everything current with the badge removal changes that are in QA, but I am getting conflicts and I think you will be better suited to solving those at this point. Thanks!

**Comment by stenington** [comment]
@adrichey I've been merging in QA, so i just merged again. The changes look correct for review.

**Comment by stenington** [comment]
And I just remembered to remove the WIP label.

**Comment by adrichey** [comment]
Reviewing now! Thanks Mike.

File: spec/features/mentor/certificates_spec.rb [hunk 1]
```diff
@@ -39,4 +39,24 @@
       href: ENV.fetch("FINALISTS_URL")
     )
   end
+
+  scenario "appreciation certificate per team" do
```
Comment (adrichey) [inline spec/features/mentor/certificates_spec.rb]: Nice additional coverage!

**Pull request merged**